templates/full-time-program.html

```diff
@@ -7,7 +7,7 @@ <h1 class="main-header__header">Full-Time Program</h1>
       <p>
         Six months of free, intensive, hands-on software engineering learning,
         followed by six months of remote job placement or job search support for women and
-        non-binary adults seeking economic empowerment.
+        non-binary adults seeking economic empowerment. The July 2024 cohort will be a five month cohort.
```

Comment:
five-month

Reply:
### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).